spec/controllers/departments_controller_spec.rb

```diff
@@ -0,0 +1,103 @@
+require 'rails_helper'
+
+RSpec.describe DepartmentsController, type: :controller do
+  let!(:department_1) { FactoryGirl.create(:department) }
+  let(:result) { JSON.parse(response.body) }
+
+  describe 'index' do
+    before { get :index }
+
+    it 'returns valid json' do
+      expect { JSON.parse(response.body) }.to_not raise_error
+    end
+
+    it 'should return a list' do
+      expect(result).to be_an(Array)
+    end
+
+    it 'includes department name and id' do
+      dept1 = result.detect { |x| x['id'] == department_1.id }
+      expect(dept1['name']).to eq(department_1.name)
+      expect(dept1['description']).to eq(department_1.description)
+    end
+
+    it 'respond succesfully' do
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  describe 'POST create' do
+    before do
+      @new_department = build(:department)
```

Comment:
I typically use a `let` block for this instead of an instance variable.  (just FYI no need to change)